intranet/apps/eighth/tests.py

```diff
@@ -428,6 +456,34 @@ def test_take_attendance_google_meet_csv(self):
         # Make sure EighthSignup object was marked absent for user2.
         self.assertTrue(EighthSignup.objects.get(user=user2, scheduled_activity=schact1).was_absent)
 
+        # Make sure bad file fails nicely with KeyError
+        with tempfile.NamedTemporaryFile(mode="w+") as f:
+            writer = csv.DictWriter(f, fieldnames=["NotName", "NotEmail"])
+            writer.writeheader()
+            writer.writerow({"NotName": "Test User", "NotEmail": "12345@fcpsschools.net"})
+            f.seek(0)
+            response = self.client.post(reverse("eighth_take_attendance", args=[schact1.id]), {"attendance": f}, follow=True)
+
+        self.assertEqual(response.status_code, 200)
```

Comment:
I think we should check for the presence of the "could not interpret file" message.

```suggestion
            self.assertIn("Could not interpret file. Did you upload a Google Meet attendance report without modification?", list(map(str, list(response.context["messages"]))))
            self.assertEqual(response.status_code, 200)
```